content/addons.md

```diff
@@ -138,20 +146,216 @@ There are several options to see the addon in action. We could use `npm link` or
+
+Let's add a class to our template and some styles to target the class:
+
+```hbs
+<!-- addon/templates/components/<addon-name>.hbs -->
+
+<button class="addon-name-button">{{yield}}</button>
+```
+
+```css
+/* addon/styles/our-addon-name.css */
+
+.addon-name-button {
+  padding: 10px;
+}
+```
+
+Now any buttons made using our addon will have the `padding: 10px` rule applied.
+
+#### CSS stylesheets that require importing from addons
+
+For some addons, it makes sense to give the developer the option to import the stylesheet we provide, or import no stylesheets at all. Using this approach, we could even offer the developer a few themes to choose from.
+
+We can do this by creating stylesheets in the `app/styles/` directory instead. These stylesheets share a file namespace with the consuming app and all the other addons someone is using, so name them wisely. For example, if we name our stylesheet `addon.css`, that's likely to clash. Just as before, it's important to choose uniquely named targets for the CSS rules so that they don't clash with other addons or the app. 
+
+Let's create `app/styles/our-addon-name.css` and add a rule to it: 
+
+```css
+/* addon/styles/our-addon-name.css */
+
+.addon-name-button {
+  border: black solid 2px;
+}
+```
+
+For the stylesheet to be active in the app the addon is used in, the developer for that app must explicitly `import` the stylesheet by name. This must be done at the very top of the app's `app.css` file.
+
+```css
+@import 'our-addon-name.css'
+/* The app's own app/styles/app.css */
+```
+
+If there are any problems getting this to work, one strategy is to build the addon with `ember build` and look inside the `dist` folder. The `dist` folder may be hidden by default in some code editors. The `dist` folder gives clues about what the consuming app sees as the file structure of the addon. See if the stylesheets are in `dist/assets/`. Then, in the Ember app, run `ember build` and look in the `dist` folder. We should see our stylesheets in `dist/assets` of the app too.
+
+#### Using CSS preprocessors for the addon's stylesheets
+
+While this guide focuses on the "out of the box" behavior of addons and the Ember CLI, there are some well-established patterns for handling stylesheets in a way that is scalable and maintainable. A CSS preprocessor like Sass (aka SCSS) allows you to nest style rules, use variables, and do simple mathematical operations.
+
+The best way to learn how to CSS preprocessors in your addon is to consult the documentation for the preprocessor addon of your choice, and study how other addon authors have implemented stylesheets. For example, [ember-styleguide](https://github.com/ember-learn/ember-styleguide/) is a UI component library that was made for the main Ember websites. It uses [ember-cli-sass](https://www.emberobserver.com/addons/ember-cli-sass) to manage styles. You can search [Ember Observer](https://emberobserver.com) for many more examples of styling in action!
```

Comment:
"learn how to CSS preprocessors" -> "learn how to use CSS preprocessors". Unless this is intentional internet-speak! =]